ci-operator/step-registry/gather/OWNERS

```diff
@@ -9,3 +9,5 @@ approvers:
 - ewolinetz
```

Comment:
Can we prune the rest of this file?  Most of them will still have approver powers via membership in the generic `ci-operator/step-registry/gather/OWNERS`, but I expect they lack specific understanding around how Loki gathering works.  I know that I'm personally lacking special Loki-gathering insight.

Reply:
Oops, you're actually touching `gather/OWNERS` here.  I think you mean to be touching `gather/loki/OWNERS`.

Reply:
ack, i'll fix this...